src/index.css

```diff
@@ -196,20 +202,3 @@ h3 {
 .pl .tracks-info .album-name {
   display: inline-block;
 }
-
-.link-disabled {
-  color: #a0a0a0;
-}
-
-.nav-links {
-  margin-top: 30px;
-  margin-bottom: 60px;
-}
-
-.nav-links a {
-  color: white;
-}
-
-.nav-links h2 {
-  margin: 10px 0px;
-}
```

Comment:
@intehon can we add these back? It's causing some weirdness with the styling of the side links

Reply:
Fixed!